services/web/server/src/simcore_service_webserver/catalog.py

```diff
@@ -55,14 +56,16 @@ async def _reverse_proxy_handler(request: web.Request):
 
     SEE https://gist.github.com/barrachri/32f865c4705f27e75d3b8530180589fb
     """
-    await check_permission(request, "services.catalog.*")
-
+    user_id = request[RQT_USERID_KEY]
```

Comment:
Is the above link still relevant?I could not figure out if the application proxy was removed.

Reply:
will check...

Reply:
@pcrespov any hint there?